applications/wg-easy/taskfiles/container.yml

```diff
@@ -1,90 +1,69 @@
 version: "3"
 
-# Development environment tasks
-tasks:
-  build-image:
-    desc: Build development container image
-    vars:
-      IMAGE_NAME: '{{.DEV_CONTAINER_REGISTRY}}/{{.DEV_CONTAINER_IMAGE}}:{{.DEV_CONTAINER_TAG}}'
-      CONTAINERFILE: '{{.CONTAINERFILE | default "./container/Containerfile"}}'
-      BUILD_ARGS: '{{.BUILD_ARGS | default ""}}'
-    requires:
-      vars: [DEV_CONTAINER_REGISTRY, DEV_CONTAINER_IMAGE, CONTAINERFILE]
+vars:
+  CONTAINER_REGISTRY: '{{.CONTAINER_REGISTRY | default "ghcr.io"}}'
+  CONTAINER_IMAGE: '{{.CONTAINER_IMAGE | default "replicatedhq/platform-examples/wg-easy-tools"}}'
+  CONTAINER_TAG: '{{.CONTAINER_TAG | default "latest"}}'
+  CONTAINER_NAME: '{{.CONTAINER_NAME | default "wg-easy-tools"}}'
+  CONTAINER_RUNTIME: '{{.CONTAINER_RUNTIME | default "podman"}}'
 
-    cmds:
-      - '{{.CONTAINER_RUNTIME}} build -t {{.IMAGE_NAME}} -f {{.CONTAINERFILE}} .'
-
-  # Start development container in background.
-  start:
-    desc: Start development container in background
+tasks:
+  exec:
+    desc: Run command in container
     silent: true
     run: once
-    vars:
-      IMAGE_NAME: '{{.DEV_CONTAINER_REGISTRY}}/{{.DEV_CONTAINER_IMAGE}}:{{.DEV_CONTAINER_TAG}}'
-      CONTAINERFILE: '{{.CONTAINERFILE | default "./container/Containerfile"}}'
-      BUILD_ARGS: '{{.BUILD_ARGS | default ""}}'
     requires:
-      vars: [DEV_CONTAINER_REGISTRY, DEV_CONTAINER_IMAGE, DEV_CONTAINER_TAG, DEV_CONTAINER_NAME, REPLICATED_API_TOKEN]
-
-    status:
-      - '{{.CONTAINER_RUNTIME}} ps | grep -q "{{.DEV_CONTAINER_NAME}}"'
+      vars: [CONTAINER_NAME, IMAGE_NAME]
+    deps:
+      - task: check-image-exists
```

Comment:
See Chris' comment https://github.com/replicatedhq/platform-examples/pull/59#discussion_r2080529669

"The security implications of this are a bit concerning, although if it's necessary it's not something I would loose sleep over.

Can you explain why kubectl port-forward is needed? Is this running port-forward from within the container, or external to the container? Is there a reason you can't just expose a port rather than use host networking?"